File: medxc-gui/src/app/model/record.ts
export class Record {
    anamnesis: string;
    observation: string;
    therapy: string;
    constructor(
        anamnesis: string,
        observation: string,
        therapy: string
    ) {
        this.anamnesis = anamnesis;
        this.observation = observation;
        this.therapy = therapy;
    }
}

File: medxc-persistence/src/main/java/com/dxc/medxc/persistence/model/Record.java
/**
 *
 */
package com.dxc.medxc.persistence.model;

/**
 * @author yyayya
 */
public interface Record {
    /**
     * @return Id of the record
     */
    int getId();

    /**
     * @return anamnesis from record
     */
    String getAnamnesis();

    /**
     * @return observation from record
     */
    String getObservation();

    /**
     * @return assigned therapy from record
     */
    String getTherapy();

    /**
     * @return appointment when the record was made
     */
    Appointment getAppointment();
}
